src/components/ReportActionItemIOUAction.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from '../ONYXKEYS';
+import ReportActionItemIOUQuote from './ReportActionItemIOUQuote';
+import ReportActionPropTypes from '../pages/home/report/ReportActionPropTypes';
+import ReportActionItemIOUPreview from './ReportActionItemIOUPreview';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+
+const propTypes = {
+    // All the data of the action
+    action: PropTypes.shape(ReportActionPropTypes).isRequired,
+
+    // The associated chatReport
+    chatReportID: PropTypes.number.isRequired,
+
+    // Should render the preview Component?
+    shouldDisplayPreviewComp: PropTypes.bool.isRequired,
```

Comment:
```suggestion
    shouldDisplayPreview: PropTypes.bool.isRequired,
```

I think we can leave out the "component" part here. Everything is a component.